src/amo/actions/reviews.js

```diff
@@ -452,11 +453,13 @@ export const setAddonReviews = ({
   pageSize,
   reviewCount,
   reviews,
+  score,
 }: SetAddonReviewsParams): SetAddonReviewsAction => {
   invariant(addonSlug, 'addonSlug is required');
   invariant(pageSize, 'pageSize is required');
   invariant(typeof reviewCount === 'number', 'reviewCount is required');
   invariant(Array.isArray(reviews), 'reviews is required and must be an array');
+  invariant(typeof score !== 'undefined', 'score is required');
```

Comment:
should this be here? I see an error on the review landing page when there is no score. I am wondering if this is causing this:

![Alt text](https://monosnap.com/image/vcAIe4DoeAW8PkPEkETxEmcRoHbBuN.png)

Reply:
> should this be here? 

yes, it should! It is catching a mistake I made in the calling code. Thanks for the screenshot, too.